Apps/WebClient/src/ClientApp/src/views/dependents.vue

```diff
@@ -120,7 +120,7 @@ export default class DependentsView extends Vue {
                         @click="showModal()"
                     >
                         <hg-icon icon="user-plus" size="medium" class="mr-2" />
-                        <span>Add a New Dependent</span>
+                        <span>Add New</span>
```

Comment:
What do you think about naming it just "Add"?

Reply:
add works for me - updating.